ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Browser:
```

Comment:
Since such an incredibly small number of our issues are caused by browser choice, do we really think it's necessary to ask for it every time?